cmd/sheltertech-go/integration_test.go

```diff
@@ -39,6 +39,28 @@ func TestGetCategoriesFeatured(t *testing.T) {
 	assert.True(t, len(categoriesResponse.Categories) > 0)
 }
 
+func TestGetCategories(t *testing.T) {
+	startServer()
+
+	url := "http://localhost:3001/api/categories"
+
+	req, err := http.NewRequest("GET", url, nil)
+	require.NoError(t, err)
+
+	res, err := http.DefaultClient.Do(req)
+	require.NoError(t, err)
+	defer res.Body.Close()
+	body, _ := ioutil.ReadAll(res.Body)
+
+	categoriesResponse := []categories.Category{}
+
+	err = json.Unmarshal(body, &categoriesResponse)
+	require.NoError(t, err)
+
+	assert.Equal(t, 121, categoriesResponse[0].Id, "12-step ID 121")
```

Comment:
I'm a bit hesitant to want to directly connect this to the specific data from the database.  It might be less error prone in the future to just check for valid structure/values exist.